md_format_tables.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/python
+
+# files to be replaced with REST-Requests
+file1 = open('table.txt')
```

Comment:
It could make sense to separate this completely from the actual logic and having the logic imported later. It would make it easier to write unit tests - for example for Marcel's edge case

Reply:
line 31 would return the md_output instead of writing it to a file

Reply:
I refactored the code accordingly.